zerver/openapi/zulip.yaml

```diff
@@ -25104,16 +25099,14 @@ components:
             A [group-setting value][setting-values] defining the set of users
             who have permission to administer this channel.
 
-            Note that a user who is a member of the specified user group must
-            also [have access](/help/channel-permissions) to the channel in
-            order to administer the channel.
+            Administrators can always administer a channel.
 
-            Realm admins are allowed to administer a channel they have access to
-            regardless of whether they are present in this group.
+            Note that a user must also [have access](/help/channel-permissions) to a
+            channel in order to modify it.
 
-            Users in this group can edit channel name and description without
-            subscribing to the channel, but they need to be subscribed to edit
-            channel permissions and add users.
+            Users can edit channel name and description without subscribing to the
+            channel, but they need to be subscribed to edit channel permissions and
+            add users.
```

Comment:
These two paragraphs might be better combined, something like:

"... in order to modify it. The exception to this rule is organization administrators can edit channel names and descriptions without having full access to the channel."

(Is that correct? Or does putting someone in the group to administer a channel allow them to see the channel exists?)

Reply:
That is correct, fixed!